bindings/python/python/opendal/__init__.pyi

```diff
@@ -15,699 +15,898 @@
 # specific language governing permissions and limitations
 # under the License.
 
+# This file is automatically generated by pyo3_stub_gen
+# ruff: noqa: E501, F401
+
+import builtins
+import collections.abc
+import datetime
 import os
-from collections.abc import AsyncIterable, Iterable
-from types import TracebackType
-from typing import TypeAlias, final
-
-try:
-    from warnings import deprecated
-except ImportError:
-    from typing_extensions import deprecated
-from opendal import exceptions as exceptions
-from opendal import layers as layers
-from opendal.__base import _Base
-from opendal.capability import Capability
-from opendal.layers import Layer
-from opendal.types import Entry, Metadata
+import pathlib
+import typing
 
-PathBuf: TypeAlias = str | os.PathLike
+from opendal import capability, exceptions, file, layers, types
```

Comment:
this import is auto generated and `capability` here was conflicting with `capability` method on Operator and AsyncOperator.

So had to change method name to `full_capability` - a user facing change.